sdk/ai/azure-ai-generative/azure/ai/generative/evaluate/_user_agent.py

```diff
@@ -0,0 +1,6 @@
+# ---------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# ---------------------------------------------------------
+from azure.ai.generative._version import VERSION
+
+USER_AGENT = "{}/{} {}/{}".format("azure-ai-generative", VERSION, "evaluate", VERSION)
```

Comment:
Nit: Can you reuse the exiting USER_AGENT and only append "evaluate" after? Having 2 versions in the string is confusing.